app/gui/vite.config.ts

```diff
@@ -57,11 +61,8 @@ export default defineConfig({
       include: fileURLToPath(new URL('./src/dashboard/**/*.tsx', import.meta.url)),
       babel: {
         plugins: [
-          '@babel/plugin-syntax-import-attributes',
-          [
-            'babel-plugin-react-compiler',
-            { target: '18', enablePreserveExistingMemoizationGuarantees: true },
-          ],
+          syntaxImportAttributes,
```

Comment:
For some reason, vitest extention fails to resolve these reds when they specified as strings (guess it tries to search them in the root of the monorepo). Using explicit import solves it

Reply:
off topic (out of scope), but i wonder why we still need import attributes? i believe they are stable in node 22, i think?

Reply:
I think no longer we need this, as it's included in the preset-env:https://babeljs.io/docs/babel-plugin-syntax-import-attributes